task/02-numbers-tasks.js

```diff
@@ -38,7 +38,7 @@ function getRectangleArea(width, height) {
  *   0    => 0
  */
 function getCicleCircumference(radius) {
-    throw new Error('Not implemented');
+    return 2 * 3.14159265358979323846264 * radius;
```

Comment:
PI is a const value, you can find it under [Math](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math) object

Reply:
fix